dev/fattest.simplicity/src/componenttest/custom/junit/runner/FATRunner.java

```diff
@@ -631,6 +631,13 @@ private String getFFDCHeader(RemoteFile ffdcLogFile) throws Exception {
                 annotationListPerClass.add(exceptionClass);
             }
         }
+        ExpectedFFDC servletFFDC = m.getMethod().getDeclaringClass().getAnnotation(ExpectedFFDC.class);
```

Comment:
I don't think `@ExpectedFFDC` will ever be used on the class level. There would have to be a scenario where every test method in a class MUST produce the same exception, which I think is extremely unlikely. Instead of adding this processing here, perhaps we could instead modify `@ExpectedFFDC` to only be applicable to methods, so that if anyone tries to put it on a non-method element in the future they would get a compile failure